controllers/cluster_controller.go

```diff
@@ -468,3 +468,12 @@ func (r *ClusterReconciler) setBundlesRef(ctx context.Context, clus *anywherev1.
 	clus.Spec.BundlesRef = mgmtCluster.Spec.BundlesRef
 	return nil
 }
+
+func getManagementCluster(ctx context.Context, clus *anywherev1.Cluster, client client.Client) (*anywherev1.Cluster, error) {
```

Comment:
is this used? we can prob remove it
also, if you look at the method above `setBundlesRef`, you might need to update it to use `FetchManagementEksaCluster`